docs/src/running-validator/validator-reqs.md

```diff
@@ -89,7 +96,7 @@ Prebuilt binaries are available for Linux x86_64 on CPUs supporting AVX2 \(Ubunt
 MacOS or WSL users may build from source.
 
 ## Networking
-Internet service should be at least 300Mbit/s symmetric, commercial. 1GBit/s preferred
+Internet service should be at least 300Mbit/s symmetric, commercial. 1GBit/s preferred. Some staked validators prefer to use 10GBit/s connections.
```

Comment:
I think we should consider saying "Internet service should be at least 1GBit/s symmetric, commercial. 10GBit/s preferred." Here